web/docs/Frontend Starters/Gatsby/Gatsby + WordPress/add-ons.md

```diff
@@ -0,0 +1,96 @@
+---
+id: 'gatsby-wordpress-add-ons'
+title: 'Add Ons'
+slug: '/frontend-starters/nextjs/gatsby/gatsby-wordpress/add-ons'
+sidebar_position: 5
+---
+
+### Advanced Custom Fields
+
+Advanced Custom Fields turns WordPress sites into a fully-fledged content
+management system by giving you all the tools to do more with your data.
+
+Advanced custom fields let you add, store, and display additional information
+about a piece of content in WordPress. This information can easily be assessed
+through GraphQL with simple adjustments to existing queries.
+
+This add-on uses GraphQL to bring in a custom `related content` data type that
+was created with Advanced Custom Fields. Your starter kit will have a related
+content section that displays at the bottom of post detail pages, if related
+posts are specified on the related WordPress post.
+
+#### Before You Begin
+
+- Install and activate the
+  [WP Advanced Custom Fields Plugin](https://wordpress.org/plugins/advanced-custom-fields/).
+
+- Install the
+  [Create Pantheon Decoupled Kit CLI](https://www.npmjs.com/package/create-pantheon-decoupled-kit/).
+
```

Comment:
"Before You Begin" should go at the top, also as an h2.

there should be no need to actually install the decoupled-kit cli. `npm create pantheon-decoupled-kit` (or pnpm create) pull the latest binary from the registry and execute it. We should probably stick with `npm create` for these docs, or mention that it will work with 'your preferred package manager (npm, pnpm, yarn)'

Reply:
@mitchellmarkoff Going back to this for a second – I am realizing this doc needs to be structured slightly differently because it will relate to multiple add-ons. I'm not sure this will be the structure we use going forward but for now I guess it makes sense.
Because of that, the `h2`s here should be the addon name and under it will be the description. Then `h3`s under that should start each section like Before You Begin, Usage, and so on. I've pushed up this change, let me know if it makes sense to you.

Reply:
Makes total sense. I was actually wondering how formatting would work when the time would come to add to this doc, I should have started that conversation earlier in the process. LGTM!